src/writer/seismogram.cpp

```diff
@@ -43,6 +43,10 @@ void specfem::writer::seismogram::write() {
                        this->output_folder + "/" + network_name + station_name +
                            "BXZ" + ".sema" };
           break;
+        case specfem::enums::seismogram::type::pressure:
+          filename = { this->output_folder + "/" + network_name + station_name +
+                       "BX" + ".semp" }; // TODO get the right name
+          break;
```

Comment:
Relevant to [this checklist](https://github.com/PrincetonUniversity/SPECFEMPP/pull/138#desc-devlist-checkbox-filename).

Reply:
The naming scheme is based off SPECFEM2D seismogram naming scheme. https://github.com/SPECFEM/specfem2d/blob/98741db1a0c8082ca57364f5b17ea95df6cbf1c2/src/specfem2D/write_seismograms.F90#L302